src/App.css

```diff
@@ -1,22 +1,73 @@
-.App {
+
+.app {
   text-align: center;
+  display: flex;
+  height: 90%;
```

Comment:
@MehwishDogar changing height does not scale the design correctly. it will affect responsiveness on different devices. you can test that thing. a good alternative is to let the content decide the height of it's container (box)